docs/photoniq/event-delivery/clients/index.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: Event Delivery Client SDKs
+---
+
+This section demonstrates some of the tasks you can complete with JavaScript SDKs. Macrometa offers the following SDKs:
+
+- [WebSocket](/download/eds-clients/photoniq-eds-ws.min.js)
+- SSE (coming soon)
+
+## Pre-requisites
+
+- Download `photoniq-eds-ws.min.js` or `photoniq-eds-sse.min.js` from section above.
+- Attach the module to your html:
+```html
+<script src="path/to/photoniq-eds-ws.min.js"></script>
+<!-- or -->
+<script src="path/to/photoniq-eds-sse.min.js"></script>
+```
+
+## Basic Example
+
+Connect to Event Delivery Service, retrieve and subsribe to SQL:
+```js
+let config = {
+    host: "<YOUR-PHOTONIQ>.photoniq.macrometa.io",
+    customerId: "<YOUR-CUSTOMER-ID>",
+    apiKey: "<YOUR-API-KEY>",
+    fabric: "<YOUR-FABRIC>",
+};
+
+let connection = PhotoniqEdsWs.connect(config);
+
+let querySet = connection.querySet();
+
+querySet.retrieveAndSubscribe("SELECT * FROM <YOUR-COLLECTION> WHERE _key='<KEY-OF-DOCUMENT>'", (event) => {
```

Comment:
Another strage is why use _key?  Why not just use "key"?

Reply:
I find the _key terrible.   My suggestion would be to remove the "_" and just call it key.

Reply:
yup

Reply:
@joubert-macrometa - I've updated the SQL.